requirements.txt

```diff
@@ -4,7 +4,7 @@ torchsde
 einops
 open-clip-torch
 transformers>=4.25.1
-safetensors
+safetensors>=0.3.0
```

Comment:
Is this to fix another issue or is this related to XPU?

Reply:
> Is this to fix another issue or is this related to XPU?

Older versions of safetensors did not support alpha version numbers of pytorch, this is fixed in the new version.
The torch version number provided with intel_extension_for_pytorch is `1.13.0a0`, and this error occurs if safetensors is not updated: https://github.com/huggingface/safetensors/issues/142